connectors/utils.py

```diff
@@ -13,7 +13,7 @@
 import re
 import shutil
 import ssl
-import subprocess
+import subprocess  # noqa S404
```

Comment:
This one is the most dangerous, please take a look at it too to make sure I did not ignore something that is super problematic

Reply:
I think it's fine and not a blocker. Looking at the code, it's only used for converting to base 64:
https://github.com/elastic/connectors/blob/68add520d325bdd000ff372dab401260b7770dad/connectors/utils.py#L256-L268
Checking git blame, it seems it has been that way since implementation, and was due to speed + less memory cost: https://github.com/elastic/connectors/pull/276

Reply:
Yeah that's what I feel too - we don't use `subprocess` for anything else.